packages/flutter_tools/test/integration.shard/vmservice_integration_test.dart

```diff
@@ -62,10 +60,10 @@ void main() {
 
     testWithoutContext('reloadSources can be called', () async {
       final VM vm = await vmService.getVM();
-      final IsolateRef isolateRef = vm.isolates.first;
+      final IsolateRef? isolateRef = vm.isolates?.first;
```

Comment:
same question, should this test be passing if we don't have the `isolateRef`? More importantly, will `vmService.callMethod()` just throw later if we pass a `null` isolateId? If so, maybe we should do `expect(isolateRef != null, true);` and then use a `!` after.

Reply:
ok